pkg/cli/predict.go

```diff
@@ -112,17 +115,18 @@ func cmdPredict(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("Invalid image name '%s'. Did you forget `-i`?", imageName)
 		}
 
-		exists, err := docker.ImageExists(ctx, imageName)
+		manifest, err := dockerCommand.Inspect(ctx, imageName)
```

Comment:
Is there a quicker way to get an existance check than an inspect call here?

Reply:
just realized you reviewed before I pushed up the changes to Pull that I mentioned. Here's the commit if you're curious: https://github.com/replicate/cog/pull/2288/commits/9e0b967cc12f1b839c7bd69ce500b341ab343d84